apps/settings/base.de.i18n

```diff
@@ -81,3 +81,4 @@ ExtAppWriteExplanation1 = "Standardmäßig externe Anwendungen"
 ExtAppWriteExplanation2 = "kann nicht in den Speicher schreiben"
 ExtAppWriteExplanation3 = "Flash (dauerhaft) Ihres Rechners."
 ExtAppEnabled = "Aufstecken"
+ClearShift = "Tastenkombination Umschalt + Löschen"
```

Comment:
I don't think shift + clear should be translated, as it's the text written on the keyboard (or French translation should follow the same translations rules as others languages to stay consistent).
```suggestion
ClearShift = "Tastenkombination shift + clear"
```